src/less/chart-legend/chart-legend.less

```diff
@@ -0,0 +1,122 @@
+.chart-legend {
+    display: inline-block;
+}
+.chart-legend--hidden {
+    border: 0;
+    clip: rect(0, 0, 0, 0);
+    height: 1px;
+    margin: -1px;
+    overflow: hidden;
+    padding: 0;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+}
+.chart-legend__table {
+    flex-shrink: 0;
+}
+.chart-legend ul {
+    list-style: none;
+    margin: 0;
+    padding-inline-start: 24px;
+}
+.chart-legend__item {
+    line-height: 2rem;
+    position: relative;
+    transition: opacity 0.5s;
+}
+.chart-legend__item--patternfill::before {
+    border-radius: 2px;
+    height: 12px;
+    position: relative;
+    top: 1px;
+    width: 12px;
+}
+.chart-legend__item--primary::before {
+    background-color: var(--data-viz-primary, var(--color-data-viz-primary));
```

Comment:
I intentionally set it to --data-viz-primary so that the donut-chart__segment--primary would be updated if it was defined as the legend and donut chart segment should match for accessibility

Reply:
So usually we want those first ones to be unique even across components. 
they aren’t used yet but they allow for customization later down the line